docker-compose.yml

```
@@ -0,0 +1,53 @@
+services:
+  odoo:
+    image: test-oca
+    networks:
+      - internal
+      - kwkhtmltopdf
+    # build:
+    #   context: ./
+    #   cache_from:
+    volumes:
+      - oca_filestore:/data/odoo/filestore
+    command: ["odoo", "-c", "/etc/odoo.cfg"]
+    ports:
+      - 8069:8069
+    environment:
+      - DB_HOST=db
+      - DB_PASSWORD=${POSTGRES_PASSWORD:-ocapwd}
+      - DB_USER=${POSTGRES_USER:-oca}
+      - DB_NAME=${POSTGRES_DB:-oca}
+      - UNACCENT=True
+      - WORKERS=2
```

```suggestion
      - WORKERS=0
```

Better for development?

---

The intent of the docker compose is mainly to use in CI to at test installing oca_all in the docker container !

It wasn't implemented when you review, any way no needs multiples odoo workers to install odoo I've followed your suggestion.